roles/common/templates/monitoring/sensu-client-path

```diff
@@ -0,0 +1,5 @@
+{% if "ceph_monitors" not in group_names -%}
+export PATH="{{ common.python.base_venv }}/bin:/opt/sensu/embedded/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/etc/sensu/plugins"
+{% else %}
+export PATH="/opt/sensu/embedded/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/etc/sensu/plugins"
```

Comment:
if we do this ...  it means any checks on ceph_monitor nodes would use system python.   We should install ceph pips into base_venv and keep it consistent ?